### ISSUE TITLE
Remove board specific settings

### ISSUE THREAD

**Issue by nseidle** [gfwilliams/svgtoeagle#10]
Thank you very much for this! I needed this to create a schematic image (a logo). To do this I removed 

* CHANGE layer tPlace; CHANGE rank 3; 
* GND signal name

from the script output. You could perhaps allow user to select the layer they want in a drop down (yay feature creep!).

I recommend adding a 3rd bubble to output format that is 'symbol'.

Again, thanks for this. Great converter.

**Comment by gfwilliams** [comment]
Hi, that's neat - I'd never considered using it for schematics!

You can change the layer you want just by typing it in... It's not a drop-down, but not the end of the world. Maybe the easiest option is just to detect if the layer name is blank, and if so to leave out `CHANGE layer tPlace; CHANGE rank 3;` (and we could do the same for the signal name as well)

**Comment by nseidle** [comment]
Yep! Polygons are less common in schematics but have a need just the same:

![image](https://user-images.githubusercontent.com/117102/53973527-288ffe00-40be-11e9-8478-dc667ec1780b.png)


Hmm - leaving those things out would work but would be a lot less obvious to those of us who don't regularly deal in ULPs and SCRs.

Again - I'm asking for the moon here, but if I selected 'symbol' it would change the layer drop down (I know, I know) to show various layers available in sch/lib and remove remove/disallow signal name.

**Comment by gfwilliams** [comment]
Ahh, I thought your name sounded familiar :) 

I'll see about adding something when I get a moment. Disabling the fields that had no effect when the sch/lib/brd was selected would be a neat addition.

If I add a dropdown it'll have to be something editable, since I bet people will want to use it on layers that they have added/renamed...

Any chance I can convince Sparkfun to start stocking our Espruino boards? ;) http://www.espruino.com

**Comment by nseidle** [comment]
Thanks! Layer selection is nice but not important in my mind (easy enough to do a select all and layer mod after import).

I've been watching Espruino since your first dev board and the boards looks very cool but JS based products have not done well for us in the past so I can't bring them in right now but I'm sure we'll meet at OHS someday :)
